examples/experimental/tongye/opacus_with_Plan/Opacus_Syft_Data_Scientist.ipynb

```diff
@@ -0,0 +1,627 @@
+{
```

Comment:
<p>Why not use real data since we have the pointer anyway? We can still limit the batches to the first 100</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/5333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>It's how you use the Plan api for the moment. When define a function as a Plan, you should give default values to it's arguments, and the defaut value should be local, serializable, and sendable. So, we don't have the data locally, and a DataLoader is not serializable and sendable for the moment (we are trying to support that in #5314). So I use a List to fake it.</p>

Reply:
<p>Yes this confusing part of the API is high on the change list. 😊</p>